packages/api/blocks-api/src/blocks/dependencies.ts

```diff
@@ -1,15 +1,2 @@
-export interface PageTreeNodeInterface {
-    id: string;
-    name: string;
-    slug: string;
-    parentId: string | null;
-}
-export interface PageTreeApi {
-    getNode: (nodeId: string) => Promise<PageTreeNodeInterface | null>;
-    getNodePath: (node: PageTreeNodeInterface) => Promise<string>;
-}
-
-export type TransformDependencies = {
-    pageTreeApi?: PageTreeApi;
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-} & Record<string, any>;
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export type TransformDependencies = Record<string, any>;
```

Comment:
Can we bring back typings when merging block into cms?

Reply:
Yes, should be possible 👍🏼